### DISCUSSION TITLE
I need help installing this extension/add-on/plugin on my Mac. I'm running Sonoma (14.7)

### DISCUSSION THREAD

**Discussion by KayScott14** [aclap-dev/video-downloadhelper#1280] [Help and Q&A]
### Your question

I need help installing this extension/add-on/plugin on my Mac. I'm running Sonoma (14.7) 
I've manually downloaded the file and tried dropping it into the Firefox extensions which did not work. I've tried opening Firefox and going to file > open file > and selecting the downloaded xpi file but that doesn't work either. Whenever I try to download it from the video download helper website it just spools, the download never initiates. I don't have pop-up block enabled or anything that should permit the download from starting. I've also downloaded it on other machines and had no issues whatsoever. 

### Addon & CoApp version and details

I need help installing this extension/add-on/plugin on my Mac. I'm running Sonoma (14.7) 
I've manually downloaded the file and tried dropping it into the Firefox extensions which did not work. I've tried opening Firefox and going to file > open file > and selecting the downloaded xpi file but that doesn't work either. Whenever I try to download it from the video download helper website it just spools, the download never initiates. I don't have pop-up block enabled or anything that should permit the download from starting. I've also downloaded it on other machines and had no issues whatsoever.

**Comment by jcfrog** [comment]
Hi,
Did you try to install it from Mozilla store? https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/
Did you install the CoApp?: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation

**Reply by KayScott14** [reply]
I figured this out, Firefox has a troubleshooting dev mode. In that mode, I went to downloads and was able to download it from there.
